webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx

```diff
@@ -66,6 +79,17 @@ export class PlayerComparePlayStyleCharts extends React.PureComponent<Props, Sta
                 label="Heatmap mode"
             />
         )
+
+        const dropDown = (
+            <PlaylistSelect
+                selectedPlaylists={[this.props.playlist]}
+                handleChange={this.handlePlaylistsChange}
+                inputLabel="Playlist"
+                helperText="Select playlist to use"
+                dropdownOnly
+                currentPlaylistsOnly
+                multiple={false}/>
```

Comment:
I think this should be better typed - if `multiple` is false, `selectedPlaylist` (no s!) takes a number and `handleChange` takes a (number => void) whereas if `multiple` is true, `selectedPlaylists` takes `number[]` and handleChange takes (number[] => void)

Reply:
I'm not sure how to do that lol

Reply:
a simple way I see to do it is 
```ts
interface MultiplePlaylistsProps {
    multiple: true
    playlists: thing[]
    handle: func
}

interface SinglePlay.. {...}

interface Props = (Single | Multiple)
 & whatever.
```